manufacturing-ai-suite/wind-turbine-anomaly-detection/docs/user-guide/Overview.md

```diff
@@ -1,87 +1,8 @@
-# Wind Turbine Anomaly Detection
+# Overview
 
-Time Series predictive maintenance use case to detect the anomalous power generation patterns relative to wind speed for the wind turbines.
+Wind Turbine Anomaly Detection use case to detect the anomalous power generation patterns relative to wind speed for the wind turbines.
 
-## Overview
-
-In the Energy Sector, such as wind turbines for power generation, unexpected equipment failures result in costly downtime and operational inefficiencies. Using AI-driven predictive analytics, edge devices can monitor equipment health through sensor data (e.g. power generation and wind speed), detect anomalous trends indicative of wear or failure, and alert operators to schedule maintenance proactively. This enhances productivity, reduces costs, and extends equipment lifespan.
+In the Energy Sector, such as wind turbines for power generation, unexpected equipment failures result in costly downtime and operational inefficiencies. Using AI-driven predictive analytics, edge devices can monitor equipment health through sensor data (For example, power generation and wind speed), detect anomalous trends indicative of wear or failure, and alert operators to schedule maintenance proactively. This enhances productivity, reduces costs, and extends equipment lifespan.
 
 This sample application demonstrates a time series use case by detecting the anomalous power generation patterns relative to wind speed. By identifying deviations, it helps optimize maintenance schedules and prevent potential turbine failures, enhancing operational efficiency. 
 
-## How it works

```

Comment:
This was required to be in Overview.md to save the package in ESC Management console tool.
Is it going to change and it is being discussed with ESC team @madhuri-rai07?